lib/fastlane/plugin/sentry/actions/sentry_upload_mobile_app.rb

```diff
@@ -0,0 +1,69 @@
+module Fastlane
+  module Actions
+    class SentryUploadMobileAppAction < Action
+      def self.run(params)
+        require 'shellwords'
+
+        Helper::SentryConfig.parse_api_params(params)
+
+        # Verify xcarchive path
+        xcarchive_path = params[:xcarchive_path]
```

Comment:
In most cases the app will be built in a prior "lane" and can be re-used automatically. Check out what we did here: https://github.com/EmergeTools/fastlane-plugin-emerge/blob/main/lib/fastlane/plugin/emerge/actions/emerge_action.rb#L17-L26

Reply:
good idea. i added this on line 42 as the default value.